pkg/terra/client/client.go

```diff
@@ -102,7 +96,7 @@ func NewClient(chainID string,
 	// Note rpchttp.New or rpchttp.NewWithTimeout use a (buggy) custom transport
 	// which results in new connections being created per request.
 	// Pass our own client here which uses a default transport and caches connections properly.
-	tmClient, err := rpchttp.NewWithClient(tendermintURL, "/websocket", &http.Client{Timeout: time.Duration(requestTimeoutSeconds)})
+	tmClient, err := rpchttp.NewWithClient(tendermintURL, "/websocket", &http.Client{Timeout: time.Duration(requestTimeoutSeconds) * time.Second})
```

Comment:
Bug fix I found testing locally, couldn't happen after this PR where the terrad tests now run in CI

Reply:
Let's just type the timeout as `time.Duration` so we avoid the issue here, and we make the constructor consistent with `FCDGasPriceEstimator` constructor. Both are used from the same place in Core so API feels odd currently.